gratka/utils.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import requests
+import unicodedata
+from scrapper_helpers.utils import caching
+try:
+    from __builtin__ import unicode
+except ImportError:
+    unicode = lambda x, *args: x
```

Comment:
seems like a hacky solution to the base problem, there must be better way of solvin that problem.